commercetools-convenience/src/test/java/io/sphere/sdk/meta/CategoryDocumentationIntegrationTest.java

```diff
@@ -299,15 +381,25 @@ private static void withProductInCategory(final BlockingSphereClient client, fin
     }
 
     private static CategoryTree fetchCurrentTree() {
-        final CompletionStage<List<Category>> categoriesStage = QueryExecutionUtils.queryAll(client(), CategoryQuery.of());
```

Comment:
When you changed the code to also provide the old API, you should be able to revert these changes.

Reply:
reverted legacy API: 16d5ea6